index.html

```diff
@@ -0,0 +1,190 @@
+            <form>
+                <p>Contact name</p>
+                <input type="text">
+                <p>Contact phone</p>
+                <input type="tel">
+                <p>Email*</p>
+                <input type="email" name="" id="">
+            </form>
+        </div>
+        <div>
+            <form>
+                <p>Let’s talk about your idea</p>
+                <textarea name="" id="" cols="30" rows="10" maxlength="320"></textarea>
+            </form>
+        </div>
+        <div>
+            <form action="">
+                <input type="range" min="0" max="100"
+                list="tickmarks">
+                <datalist id="tickmarks">
+                <option value="0"></option>
+                <option value="10"></option>
+                <option value="20"></option>
+                <option value="30"></option>
+                <option value="40"></option>
+                <option value="50"></option>
+                <option value="60"></option>
+                <option value="70"></option>
+                <option value="80"></option>
+                <option value="90"></option>
+                <option value="100"></option>
+                </datalist>
+            </form>
+        </div>
+        <hr>
+        <div>
+            <img src="img/upload.png" alt="button_upload">
+            <a href=""> Upload Additional file</a>
+            <p>File size of your documents should not exceed 10MB</p>
+        </div>
+        <div>
+            <button>SUBMIT</button>
+            <input type="radio" name="" id="">I want to protect my data by signing an NDA
+        </div>
+        <div>
+            <form>  
+                <input type="checkbox" name="" id="">I want to protect my data by signing an NDA
+                <input type="checkbox" name="" id=""checked>
+                <span>I have read and accept the Terms and Conditions</span>
+            </form>
+        </div>
+    </main>    
+    <footer>
+        <div> 
+            <div>
+                <div>
+                    <h2>Career</h2>
+                </div>
+                <div>
+                    <p>If you are interested in working at Logo send us your CV to <a href="careers@logo.com">careers@logo.com</a> and we can talk</p>
+                </div>
+            </div>
+            <div>
+                <a href="https://www.instagram.com/">
+                    <img src="img/instagram.png" alt="instagram">
+                </a>
+                <a href="https://www.facebook.com/">
+                    <img src="img/facebook.png" alt="facebook">
+                </a>
+                <a href="https://telegram.org/">
+                    <img src="img/telegram.png" alt="telegram">
+                </a>
+            </div>
+        </div>
+        <div>      
+            <div>
+                <h2>Offices</h2>
+            </div>
+            <div>
+                <p>United States of America<br>500 5th Avenue Suite 400, NY 10110</p>

```

Comment:
It's not a mistake. Just a suggestion to make html more semantic: what about using `<address>` tag?
https://developer.mozilla.org/ru/docs/Web/HTML/Element/address